src/lib/components/forms/form/form-components/multi-select/multi-select.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, Input } from '@angular/core';
+import { TerraPlacementEnum } from '../../../../../helpers/enums/terra-placement.enum';
+import { TerraSelectBoxValueInterface } from '../../../select-box/data/terra-select-box.interface';
+import { noop } from 'rxjs';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { MultiSelectInterface } from './multi-select.interface';
+
+@Component({
+    selector: 'tc-multi-select',
+    templateUrl: './multi-select.component.html',
+    providers: [
+        {
+            provide: NG_VALUE_ACCESSOR,
+            useExisting: MultiSelectComponent,
+            multi: true
+        }
+    ]
+})
+export class MultiSelectComponent implements ControlValueAccessor, MultiSelectInterface {
+    /** The name of the control which will be used as label. */
+    @Input()
+    public name: string = '';
+
+    /** Disables the input when set to true. Default false. */
+    @Input()
+    public isDisabled: boolean = false;
+
+    /** Requires the input to be filled when set to true. Default false. */
+    @Input()
+    public isRequired: boolean = false;
+
+    /** Text that should be shown in a tooltip on the control. */
+    @Input()
+    public tooltipText: string;
+
+    /** Set the tooltip placement (bottom, top, left, right). Default top. */
+    @Input()
+    public tooltipPlacement: TerraPlacementEnum = TerraPlacementEnum.TOP;
+
+    /** List of options that the user can select from. */
+    @Input()
+    public listBoxValues: Array<TerraSelectBoxValueInterface>;
```

Comment:
Sollte das nicht `TerraMultiCheckBoxValueInterface` sein? Das hat die Property `selected` . Ist jedoch genauso deprecated wie das TerraSelectBoxValueInterface :D

Reply:
stimmt! ist auch deprecated aber ist ja sowieso 'for internal use'